solutions/sandbox-gke/Kptfile

```diff
@@ -1,16 +0,0 @@
-apiVersion: kpt.dev/v1
```

Comment:
if there's any value/reusability to the package, should it be moved in the legacy folder instead of deleting?
otherwise, deleting is fine

Reply:
I think it's out of date enough that it still would require a bit of work to get up to the latest specs to get up to date. I thought about that but I don't feel there's enough value there any more.